### ISSUE TITLE
Write concordances as components of the CLDF dataset

### ISSUE THREAD

**Issue by xrotwang** [cldf/pyigt#3]
It would be nice if the output created by `pyigt` could be fed back directly into the CLDF dataset. It's unclear, though, which intermediate outputs would make sense, rather than going for a full CLDF Dictionary.

**Comment by LinguList** [comment]
Yes, that is something we need to address in the future. Ideally, the dictionary would give IDs for word families, so they could be referenced consistently in IGT, right? But that won't happen, unless somebody does it, so we may have to wait for the time being here.
